framework/src/modules/dpos/transaction_assets/pom_transaction_asset.ts

```diff
@@ -230,10 +230,16 @@ export class PomTransactionAsset extends BaseAsset<PomTransactionAssetContext> {
 		const delegateAccountBalance = await reducerHandler.invoke<bigint>('token:getBalance', {
 			address: delegateAccount.address,
 		});
+		const minRemainingBalance = await reducerHandler.invoke<bigint>('token:getMinRemainingBalance');
+
+		const delegateSubtractableBalance =
+			delegateAccountBalance - minRemainingBalance > BigInt(0)
```

Comment:
Perhaps this could be better as a regular if as there's some calculation going on instead of straight selection of values based on input. But is just opinion so ok to keep.

Reply:
It might be slightly better and it looks good, but ill just keep it as it is since it doesnt change significantly in both way